SECURITY.md

```diff
@@ -0,0 +1,37 @@
+# Security Policy
+
+## Supported Versions
+
+The latest versions of INDIGO IAM project is currently being supported with security updates.
+At the administrators' discretion, security patches may be considered for versions other than the latest.
+
+## Reporting a Vulnerability
+
+This repository has a private vulnerability reporting enabled.
+This makes it easy for security researchers to report vulnerabilities directly to the repository maintainers using a simple form.
+
+Follow the following steps if you need to report a vulnerability or a security issue:
+
+1. On GitHub, navigate to the [main page](https://github.com/indigo-iam/iam) of the repository 
+2. Under the repository name, click  [Security](https://github.com/indigo-iam/iam/security). If you cannot see the "Security" tab, select the  dropdown menu, and then click Security.
+3. Click [Report a vulnerability](https://github.com/indigo-iam/iam/security/advisories/new) to open the advisory form.
+4. Fill in the advisory details form.
+
+> [!TIP]
+> In this form, only the title and description are mandatory.
+> However, we recommend provide as much information as possible on the form so that the maintainers can make an informed decision about the submitted report.
```

Comment:
"we recommend to provide.."

Reply:
I think `recommend` is followed by a _gerund_ ("we recommend providing") or with _that_ clause ("we recommend that you provide").